repository/dependencies.json

```diff
@@ -4,1357 +4,1418 @@
 	"dependencies": [
 		{
 			"name": "anytree",
-			"load_order": "01",
 			"description": "Python Anytree module - https://github.com/c0fec0de/anytree",
 			"author": "c0fec0de",
 			"issues": "https://github.com/c0fec0de/anytree/issues",
+			"load_order": "01",
 			"releases": [
 				{
-					"sublime_text": "*",
 					"base": "https://github.com/nbeversl/anytree_sublime",
+					"platforms": ["*"],
```

Comment:
Was there a particular reason for adding this to all entries where it was missing? E.g. considerations for ARM architectures and changing the default in 4.0.0?

Reply:
Just want to have all entries to be of equal structure as it makes adding `python_versions` easier. We could also remove it from all, wich only contain asterisks, but I like explicity in general.

Reply:
The issue I have with it is that if we want it to be explicit, we also need to enforce it (in the tests), otherwise the next PR adding a dependency might not use it and we're back to where we started.

That said, I generally prefer reasonable defaults that do not need to be modified and not having to specify a platforms key to mean "install for all" seems like a reasonable default that reduces noise in the actual file.

Reply:
So `"sublime_text": "*"` would also need to be stripped everywhere, too, as it is also a reasonable default. This PR is an attempt to introduce a common structure (used keys and their order) in at least libraries.json.

Expilicit keys were choosen as it is what will be needed for `"python_versions"` in scheme_version 4.0.0. It must list supported python versions explicitly. So it made sense to me to treat the other parts the same.

see: https://github.com/deathaxe/package_control_channel/blob/four-point-oh/repository/libraries.json

Order and presence of those wildcard keys is a mess across the repo.

Reply:
Yes, `sublime_text` is required because we request package authors and dependency authors to make a conscience choices about that, especially after ST3 was introduced with different Python versions and supporting both at the same time was not the norm. The same also holds true for the `python_versions` key, so I'm definitely on board with requiring that.

Now, I presume that `sublime_text` will be automatically softly restricted to something above 3000 due to the PC update not being available for ST2, otherwise `"sublime_text": "*", "python_versions": ["3.3"]` wouldn't make sense.

As for platforms, I still believe the default of allowing for all platforms is sensible, but seeing how basically every other key is required for dependencies I'm not strongly tied up on it.